notebook/static/tree/js/notebooklist.js

```diff
@@ -315,6 +316,35 @@ define([
         this._selection_changed();  
     };
 
+    NotebookList.prototype.is_selected = function(){
```

Comment:
This isn't really `is_selected` since there is a side effect to focus on the first list item if nothing is in focus. Where is this function used?